phpstan-baseline.neon

```diff
@@ -59750,6 +57740,11 @@ parameters:
 			count: 1
 			path: src/Sulu/Component/Webspace/Settings/SettingsManager.php
 
+		-
+			message: "#^Method Sulu\\\\Component\\\\Webspace\\\\Settings\\\\SettingsManager\\:\\:loadStringByWildcard\\(\\) should return array\\<string\\> but returns array\\<string, array\\|string\\>\\.$#"
```

Comment:
did i get something wrong with anntations? or is this not related to phpcr changes?

Reply:
Not sure how `getString` can be `array|string` here: https://github.com/sulu/sulu/blob/1fc46655a4a14afbb2600b6524ff63153d5fd4e1/src/Sulu/Component/Webspace/Settings/SettingsManager.php#L117

Reply:
Looks like an error on our side not handling `multivalues` if I understand correctly: https://github.com/phpcr/phpcr/blob/3307bbd639408f16fd947fc1be1f9cf3410c52a6/src/PHPCR/PropertyInterface.php#L575-L576

Reply:
Probably should in phpcr be then `string|string[]`

Reply:
there is a lot of places that should be that. fixed them now.